plugins/com.google.cloud.tools.eclipse.appengine.compat/src/com/google/cloud/tools/eclipse/appengine/compat/gpe/GpeConvertCommandHandler.java

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
copyright

Reply:
Weird. My settings are configured to produce the header, and it works sometimes.